client/lib/highlight/index.js

```diff
@@ -72,7 +72,7 @@ function walk( node, term, wrapperNode ) {
 	let children;
 	debug( 'Node type', node.nodeName );
 	if ( node.childNodes.length ) {
-		children = toArray( node.childNodes );
+		children = Array.from( node.childNodes );
```

Comment:
This one shouldn't be needed. `childNodes.length` and `childNodes[ i ]` are natively supported.

Reply:
Good point. Addressed in 9394e754a8650e50ae4154481a837e88374322fc and removed the `children` variable completely.